### PR TITLE
Created body

### PR DESCRIPTION
**Pull request by idungstanley**
Add the body of the portfolio including the about me section, and the project template showing different projects and skills

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -63,6 +61,161 @@ <h2>I’am a Software Developer</h2>
...
+            sign-ups required. has been the industry's standard
+          </p>
+          <ul class="list-container">
+            <li class="list-2"><a href="">html</a></li>
+            <li class="list-2"><a href="">bootstrap</a></li>
+            <li class="list-2"><a href="">Ruby</a></li>
+          </ul>
+        </article>
+        <button class="base-btn">See Project</button>
+      </section>
```
Comment (karvel-code) [inline index.html:165]: - [ ] It is commendable that you have styled this section using `display: flex`. However, the requirements specify that you use `display: grid`. Please change that. (This is just for the main container of the projects section. The individual projects can be styled using flexbox)

**Review by karvel-code — CHANGES_REQUESTED** [review]
Hi @idungstanley , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- [x] You have a very professional README. Well done 👏 👏 
- [ ] Your linter checks are passing sucessfully.
- [ ] You have clean and organized code 💯 .


## Status: Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- n/a

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (idungstanley) [reply index.html:165]: I have added the grid layout as recommended.

**Review by aimemalaika — APPROVED** [review]
Hi @idungstanley , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

133591850-fdcb7dd3-dd7a-4274-81e7-0e59434148f7

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**